src/main/resources/containers.json

```diff
@@ -1,1781 +1,2402 @@
-            }
-        ],
-
-        "download": {
-            "group_id": "org.wildfly",
-            "artifact_id": "wildfly-dist"
-        }
-
-
-    },
-    {
-        "artifact_id": "jboss-as-arquillian-container-remote",
-        "group_id": "org.jboss.as",
-        "name": "Arquillian Container JBoss AS Remote 7.x",
-        "container_type": "REMOTE",
-        "configurations": [
-            {
-                "type": "java.lang.String",
-                "description": "The ip address of the running server instances management interface",
-                "name": "managementAddress",
-                "default": "127.0.0.1"
-            },
-            {
-                "type": "java.lang.Integer",
-                "description": "The management port for deployment",
-                "name": "managementPort",
-                "default": "9999"
-            }
-        ]
-    },
-    {
-        "artifact_id": "wildfly-arquillian-container-remote",
-        "group_id": "org.wildfly",
-        "name": "Arquillian Container WildFly Remote 8.x",
-        "container_type": "REMOTE",
-        "configurations": [
-            {
-                "type": "java.lang.String",
-                "description": "The ip address of the running server instances management interface",
-                "name": "managementAddress",
-                "default": "127.0.0.1"
-            },
-            {
-                "type": "java.lang.Integer",
-                "description": "The management port for deployment",
-                "name": "managementPort",
-                "default": "9999"
-            }
-        ]
-    },
-    {
-        "artifact_id": "arquillian-weld-ee-embedded-1.1",
-        "group_id": "org.jboss.arquillian.container",
-        "name": "Arquillian Container Weld EE Embedded 1.1",
-        "container_type": "EMBEDDED",
-        "configurations": [
-            {
-                "type": "java.lang.Boolean",
-                "description": "Activate ConversationScope between @Test methods. Use this to simulate Weld Servlet HTTP Conversation scope support.",
-                "name": "enableConversationScope",
-                "default": "false"
-            }
-        ],
-        "dependencies": [
-            {
-                "artifact_id": "weld-core",
-                "group_id": "org.jboss.weld"
-            },
-            {
-                "artifact_id": "weld-api",
-                "group_id": "org.jboss.weld"
-            },
-            {
-                "artifact_id": "slf4j-simple",
-                "group_id": "org.slf4j"
-            }
-        ]
-
-    },
-    {
-        "artifact_id": "arquillian-weld-se-embedded-1",
-        "group_id": "org.jboss.arquillian.container",
-        "name": "Arquillian Container Weld SE Embedded 1.0",
-        "container_type": "EMBEDDED"
-    },
-    {
-        "artifact_id": "arquillian-weld-se-embedded-1.1",
-        "group_id": "org.jboss.arquillian.container",
```

Comment:
Yes, we have containers supported by chameleon here. As we couldn't call some logic from chameleon-model which unnecessarily force me to add this group ID, artifact ID here. 

If we parse yaml file line by line then we have some Reference variable which is causing problem. 

We can add one issue to enhance it such that it will use from chameleon default yaml file.